tests/struct.rs

```diff
@@ -64,11 +74,20 @@ fn struct_test_cases() -> Vec<TestCase<StructTestCases>> {
     ];
     b1_expected.reverse();
 
+    let c0 = C {
+        y: CInner {},
+        z: None,
+    };
+
+    let mut c0_expected = vec![0x00, 0x00, 0x01, 0x01];
```

Comment:
I'm confused why this is the expected output. Surely it should be just `[0x00]` since the only thing that carries data is the `None`. Guess there is something we don't understand about serializing empty structs